mcconf.py

```diff
@@ -355,7 +355,9 @@ def resolveDependencies(self):
                 mod = solutions.pop()
                 # 2) ignore if conflict with already selected modules
                 conflicts = self.provides & mod.provides
-                if conflicts: continue
+                if conflicts:
+                    logging.debug('Dropped dependency %s to %s because of conflicting provides %s', tag, mod.name, conflicts)
```

Comment:
The log message is very hard to interpret without reading the code. If I get it right, the dependency is ignored because the module that provides the dependency provides other dependencies that are already satisfied.

Maybe this is more clear?
```python
logging.debug('Dropped dependency  %s to %s because of a conflict with already selected modules providing %s', tag, mod.name, conflicts)
```